api/libproject/devstart.rb

```diff
@@ -634,6 +634,116 @@ def run_local_all_migrations()
   :fn => ->() { run_local_all_migrations() }
 })
 
+def liquibase_gradlew_command(verification_command, argument, run_list)
+  verificion_full_cmd = %W{../gradlew #{verification_command} -PrunList=#{run_list}}
+  unless argument.nil? || argument.empty?
+    verificion_full_cmd << argument
+  end
+  verificion_full_cmd
+end
+
+# Run a liqibase command against the specified project. Where possible, show SQL
+# statements and ask user for verification.
+def run_liquibase(cmd_name, *args)
+  command_to_sql = {
+      'changelogSync' => 'changelogSyncSQL',
+      'markNextChangesetRan' => 'markNextChangesetRanSQL',
+      'rollback' => 'rollbackSQL',
+      'rollbackCount' => 'rollbackCountSQL',
+      'rollbackToDate' => 'rollbackToDateSQL',
+      'update' => 'updateSQL',
+      'updateCount' => 'updateCountSql',
+      'updateToTag' => 'updateToTagSQL'
+  }
+
+  common = Common.new
+  ensure_docker(cmd_name, args)
+
+  op = WbOptionsParser.new(cmd_name, args)
+  op.add_typed_option(
+      "--command [command]",
+      String,
+      ->(opts, c) { opts.command = c},
+      "Liquibase command, e.g. update, rollback, tag, validate. See "+
+          "https://www.liquibase.org/documentation/command_line.html")
+  op.add_typed_option(
+      "--argument [argument]",
+      String,
+      ->(opts, a) { opts.argument = a},
+      "Liquibase command argument, e.g. count or tag value")
+  op.add_typed_option(
+        "--run-list [run_list]",
+        String,
+        ->(opts, rl) { opts.run_list = rl },
+        "Liquibase runList, a comma-separated list of activities in the liquibase task")
+  op.add_typed_option(
+        '--project [project]',
+        String,
+        ->(opts, p) { opts.project = p },
+        'AoU environment GCP project full name. Used to pick MySQL instance & creadentials.'
+  )
+  op.add_validator ->(opts) {
+    if opts.command.nil? || opts.command.empty?
+      raise ArgumentError.new("command is required")
+    end
+  }
+  op.parse.validate
+
+  # Currently there's only one activity (main), and leaving out the runList arugment causes
+  # it to run that activity. However, that's not because it's detected as default or primary, but
+  # leaving out the runList is equivalent to specifying all of the activities to run in unspecified
+  # order.
+  # https://github.com/liquibase/liquibase-gradle-plugin#3-configuring-the-plugin
+  if op.opts.run_list.nil? || op.opts.run_list.empty?
+    run_list = "main"
```

Comment:
Does this duplicate the effect that `gradle.properties` is having? I assume that's just setting a default

Reply:
Oh good point. I just added the gradle.properties to shut up the IJ plugin so it would let me add `db` as a gradle project. Let me play with this.